src/wasm/wasm-s-parser.cpp

```diff
@@ -2706,6 +2706,24 @@ Expression* SExpressionWasmBuilder::makeTableFill(Element& s) {
   return Builder(wasm).makeTableFill(tableName, dest, value, size);
 }
 
+Expression* SExpressionWasmBuilder::makeTableCopy(Element& s) {
+  auto destTableName = s[1]->str();
```

Comment:
Should we support raw indices as well, or do we not support those in other places? The new wat parser will support them in either case (eventually).

Reply:
I don't think we do in other places, as this is based on TableFill right above us. That actually might explain why some spec tests didn't pass...